force-app/main/default/classes/ClassesObjectsMethods.cls

```diff
@@ -190,15 +178,19 @@ public with sharing class ClassesObjectsMethods {
      * @return The number of the day of the week for the given date.
      */
     public static String getDayOfWeek(DateTime dt) {
-        // Define a map of day of the week as a key and day number as the value
-        // Monday = 1, Tuesday = 2, etc.
-
-        // Get the day name using .format('EEEE');
-
-        // Get the day number from the map using the day name
-
-        // Return the result as a string
-        return null; // Replace null with the variable you used to store the result
+        Map<String, Integer> daysOfWeek = new Map<String, Integer>();
+        daysOfWeek.put('Monday', 1);      // Define a map of day of the week as a key and day number as the value
+        daysOfWeek.put('Tuesday', 2);  
+        daysOfWeek.put('Wednesday', 3);
+        daysOfWeek.put('Thursday', 4);
+        daysOfWeek.put('Friday', 5);
+        daysOfWeek.put('Saturday', 6);
+        daysOfWeek.put('Sunday', 7);
+
```

Comment:
The `put()` method is great when building a map bit by bit (especially in a loop). But if you know exactly what's going to be in your map from the start, you can build it like this:
```suggestion
        Map<String, Integer> daysOfWeekMap = new Map<String, Integer>{
            'Monday' => 1,
            'Tuesday' => 2,
            'Wednesday' => 3,
            'Thursday' => 4,
            'Friday' => 5,
            'Saturday' => 6,
            'Sunday' => 7
        };

```
I also like to put `Map` in the name of my map variable so it's clear what it represents.